### DISCUSSION TITLE
节点端口反复被封

### DISCUSSION THREAD

**Discussion by tedestar** [v2fly/v2ray-core#1952] [General]
跟本地ISP有关，访问google之后，本地的ISP网络就无法访问节点的服务端口了（同一台电脑，我用电信网络出现这个问题，但手机网络没这个问题，显而易见跟电信的ISP有关）

就是只要连上google，这个ISP就封禁这个端口，换个端口可以了，然后再来一次，都是瞬间直接封。

不知道这个原理是什么？难道是根据DNS来的吗？

**Comment by kulongwangzhi85** [comment]
配置不上来看看？

**Comment by tedestar** [comment]
不知道这个配置对不对
{
	"outbounds": [
		{
			"protocol": "vmess",
			"settings": {
				"vnext": [
					{
						"address": "47.*.*.*",
						"port": 2001,
						"users": [
							{
								"id": "f408d168-****-47ad-9b39-*********",
								"alterId": 0,
								"security": "auto"
							}
						]
					}
				]
			},
			"mux": {
				"enabled": true
			}
		},
		{
			"protocol": "freedom",
			"settings": {},
			"tag": "direct"
		},
		{
			"protocol": "blackhole",
			"settings": {},
			"tag": "blocked"
		},
		{
			"protocol": "dns",
			"tag": "dns-out"
		}
	],
	"inbounds": [
		{
			"port": "1099",
			"protocol": "dokodemo-door",
			"settings": {
				"network": "tcp,udp",
				"timeout": 0,
				"followRedirect": true
			},
			"sniffing": {
				"enabled": true,
				"destOverride": [
					"http",
					"tls"
				]
			}
		},
		{
			"port": 2133,
			"tag": "dns-in",
			"protocol": "dokodemo-door",
			"settings": {
				"address": "119.29.29.29",
				"port": 53,
				"timeout": 0,
				"network": "tcp,udp"
			}
		},
		{
			"port": 2333,
			"protocol": "socks",
			"settings": {
				"auth": "noauth",
				"udp": true
			}
		},
		{
			"port": 6666,
			"protocol": "http",
			"settings": {
				"auth": "noauth",
				"udp": true
			}
		}
	],
	"dns": {
		"servers": [
			{
				"address": "119.29.29.29",
				"port": 53,
				"domains": [
					"geosite:cn"
				],
				"expectIPs": [
					"geoip:cn"
				]
			},
			{
				"address": "1.1.1.1",
				"port": 53,
				"domains": [
					"geosite:geolocation-!cn"
				]
			},
			"8.8.8.8",
			"localhost"
		]
	},
	"routing": {
		"domainStrategy": "IPOnDemand",
		"rules": [
			{
				"type": "field",
				"inboundTag": [
					"dns-in"
				],
				"outboundTag": "dns-out"
			},
			{
				"type": "field",
				"ip": [
					"geoip:private"
				],
				"outboundTag": "blocked"
			},
			{
				"type": "field",
				"ip": [
					"geoip:cn"
				],
				"outboundTag": "direct"
			},
			{
				"type": "field",
				"domain": [
					"geosite:cn"
				],
				"outboundTag": "direct"
			}
		]
	}
}

**Reply by meikeeit** [reply]
好家伙，裸vmess啊，你才是真正的猛士

**Comment by kulongwangzhi85** [comment]
裸奔加使用不知名端口，vps的配置呢？配置不当，导致流量回流国内也是很大的问题。
